signac/contrib/project.py

```diff
@@ -730,6 +745,8 @@ def _get_statepoint_from_workspace(self, jobid):
                 raise JobsCorruptedError([jobid])
             raise KeyError(jobid)
 
+    @deprecated(deprecated_in="1.3", removed_in="2.0", current_version=__version__,
+                details="Use open_job(id=jobid).statepoint() function instead.")
```

Comment:
This function is used internally, so unless this is refactored, we should probably move its logic into a private method. But we could (should) do that as part of a separate refactoring PR.

Reply:
Created #227.